lib/loaders/express.loader.ts

```diff
@@ -48,6 +48,14 @@ export class ExpressLoader extends AbstractLoader {
         app.use(express.static(clientPath, options.serveStaticOptions));
         app.get(options.renderPath, renderFn);
       }
+
+      app.use((err: Error, req, res, next: Function) => {
```

Comment:
@micaelmbagira  I just learned this recently, but you `express` exports a type call `NextFunction` specifically for the next parameter. Might be better to use that instead.

Reply:
Not sure if that would be useful in that case. We need to send the 404 to the client and I don't see any other way to do it

Reply:
No your solution is good. I am just suggesting a narrower type on `next`. Right now we are using `Function`, but we could use https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express/index.d.ts#L107. It would help narrow the typing down a bit :). Unless there is something that I am missing.